lib/c-api/Cargo.toml

```diff
@@ -70,14 +71,18 @@ llvm = [
     "compiler",
 ]
 
+# Include the deprecated Wasmer C API.
+deprecated = []
+
+# Deprecated feature.
+# TODO: Port this feature.
 #emscripten = ["wasmer-emscripten"]
-# used to avoid generating standard Wasm C API types in our header files
-ignore-wasm-c-api = []
 
 # This is for compatibility for old usage
 singlepass-backend = ["singlepass"]
 cranelift-backend = ["cranelift"]
 llvm-backend = ["llvm"]
 
 [build-dependencies]
-cbindgen = { version = "0.14.3" }
+#cbindgen = "0.14.6"
+cbindgen = { git = "https://github.com/eqrion/cbindgen", branch = "master" }
```

Comment:
What if we target a specific commit?

Reply:
That won't help with publishing to crates.io, but I think we shouldn't block merging this on that just because of how much trouble it would be to delay it with other changes to the C API...

If it's on master then we can ask for another release and hopefully get one soon!